install/scripts/php-conf-fix.sh

```diff
@@ -6,7 +6,24 @@
 #
 echo 'Automated php.ini configuration adjustments'
 echo
-phpIni=/etc/php5/apache2/php.ini
+if [ -f /etc/debian_version ]
+then
+    phpIni=/etc/php5/apache2/php.ini
+    TIMEZONE=`cat /etc/timezone`
+    if [ -z $TIMEZONE ]
+    then
+        TIMEZONE="America/Denver"
+    fi
+elif [ -f /etc/redhat-release ]
+then
+    phpIni=/etc/php.ini
+    TIMEZONE=`readlink /etc/localtime | sed 's/.*\usr\/share\/zoneinfo\///'`
```

Comment:
There is missing the backslash in front of `usr`

Even simpler: you can replace the delimiter by `%` and write the command without escaping as 
```
    ... | sed 's%/usr/share/zoneinfo/%%'`
```